neurom/apps/morph_stats.py

```diff
@@ -164,36 +162,36 @@ def extract_stats(neurons, config):
     {config_path}
     """
 
-    def _fill_stats_dict(data, stat_name, stat):
+    def _fill_stats_dict(data, stat_name, stat, shape):
         """Insert the stat data in the dict.
 
-        And if the stats is a 3D array, splits it into XYZ components.
+        If the feature is 2-dimensional, the feature is flattened on its last axis
         """
-        if stat is None or not isinstance(stat, np.ndarray) or stat.shape not in ((3, ), ):
-            data[stat_name] = stat
+        assert len(shape) <= 2, f'Wrong shape: {shape}'
```

Comment:
We still need to agree on that, but I'd prefer to avoid the assert and add:
```
elif len(shape) > 2:
    raise ValueError(f'Feature with wrong shape: {shape}')
```